langwatch_nlp/langwatch_nlp/studio/execute/execute_evaluation.py

```
@@ -30,6 +30,11 @@
 from dspy.utils.asyncify import asyncify
 from sklearn.model_selection import train_test_split
 
+from langwatch.batch_evaluation import DatasetRecord, DatasetEntry
```

Ruff (0.11.9)

`langwatch.batch_evaluation.DatasetRecord` imported but unused

Remove unused import

(F401)